velox/exec/RowContainer.h

```diff
@@ -302,6 +302,13 @@ class RowContainer {
       char* row,
       int32_t columnIndex);
 
+  /// Stores the 'decoded' into row container at 'column'.
+  void storeVector(
+      const DecodedVector& decoded,
+      const SelectivityVector* rows,
```

Comment:
Why 'rows' is a pointer and not a reference? Do we allow 'null' rows? If so, let's document this and make sure to test.

That said, if there is no use case for 'rows' where not all rows are selected, let's pass 'size' instead. This is simpler and requires less testing. Furthermore, passing SelectivityVector requires explaining how 'groups' are indexed. Do we store each 'row' in groups[row] or do we store selected rows in groups consecutively.

Finally, 'groups' is not the right name for this parameter. it should be 'rows'. I see how it conflicts with SelectivityVector 'rows'. Once we change it back to 'size' the conflict will go away.

Reply:
Updated in the new PR, thanks.